tests/fixtures/basket_ball.json

```diff
@@ -7,7 +7,20 @@
     "label": "62B Rue de Lille (Paris)",
     "name": "62B Rue de Lille",
     "postcode": "75007",
-    "street": "Rue de Lille",
```

Comment:
We should check these fixtures. I don't think that `postcode` should appear in the documents that are inserted into ES.